scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -242,9 +247,21 @@ case class Args(
     RuleDecoder.decoder(ruleDecoderSettings.withConfig(scalafixConfig))
   }
 
+  // With a --on-compile flag, looking for settings in onCompile block first, and fallback to standard settings.
+  def preProcessedConf(base: Conf): Conf =
```

Comment:
if we use the `overlay` term, this could be more precise
```suggestion
  def maybeOnCompileOverlaidConf(base: Conf): Conf =
```

Reply:
`Overlay` seems to be a nicer word 👍